### PR TITLE
Cleanup deps

### PR DESCRIPTION
**Pull request by wouterj**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -34,8 +31,7 @@
         "doctrine/phpcr-odm": "To enable support for the PHPCR ODM documents (^1.4)",
         "doctrine/phpcr-bundle": "To enable support for the PHPCR ODM documents",
         "doctrine/orm": "To enable support for the ORM entities (^2.5)",
-        "symfony-cmf/content-bundle": "To optionally use the configured value for 'content_basepath' from the CoreBundle",
-        "symfony-cmf/core-bundle": "To use the RouteTypeType form type and for easier configuration (^2.0)"
```
Comment (dbu) [inline composer.json]: are you aware that this form type exists and relies on something in core bundle?

Comment (wouterj) [inline composer.json]: I am aware that this form type exists, but I can't find how it's related to the CoreBundle: https://github.com/symfony-cmf/routing-bundle/blob/master/src/Form/Type/RouteTypeType.php

Comment (dbu) [inline composer.json]: i read "core" in the namespace of one of the use statements. but its core of symfony component. sorry

Comment (wouterj) [inline composer.json]: No problem :)

(Btw can you please check the Packagist issue of corebundle?)

**Pull request merged**